static/src/stylesheets/module/facia-garnett/item-types/_fc-item--type-liveblog.scss

```diff
@@ -0,0 +1,61 @@
+.fc-container:not(.fc-container--story-package) .fc-item--type-liveblog {
```

Comment:
We totally don't want these styles to override story package styling. I also don't think I want to re-override these styles with  story package styles later in this module, i.e.
```scss
.fc-item--type-liveblog {

    //...all the live card styling, and then...

    .fc-container--story-package & {
        .fc-item__kicker {
            background-color: $news-support-7;
            color: $live-garnett-main-1;
        }

        .u-faux-block-link--hover .fc-item__kicker {
            background-color: darken($news-support-7, 5%);
        }

        .fc-trail__count--commentcount {
            color: #dadcdd;

            .inline-icon {
                fill: #dadcdd;
            }
        }
    }
```

Reply:
Handy that there's a liveblog in a story-package container today! :sweat_smile:

Out of interest why not override the default liveblog styles as per you're example? Do you foresee the story package overrides sitting somewhere else?

Reply:
I know, right!

It's more a structural decision, and a minor performance consideration. Story package styles already override facia cards. So the live blog card styles partially override the story package overrides, then my example code above overrides those overrides. So some of the elements will have their styles overridden four times, which seemed a bit excessive.

Perhaps I'm being a bit pernickety?